src/content-handlers/iiif/modules/uv-shared-module/BaseExtension.ts

```diff
@@ -765,13 +765,8 @@ export class BaseExtension<T extends BaseConfig> implements IExtension {
   }
 
   getAppUri(): string {
-    const appUri: string =
-      window.location.protocol +
-      "//" +
-      window.location.hostname +
-      (window.location.port ? ":" + window.location.port : "");
-
-    return appUri + "/uv.html";
+    const port = window.location.port ? `:${window.location.port}` : "";
+    return `${window.location.protocol}//${window.location.hostname}${port}`;
```

Comment:
Rather than moving `/uv.html` from here into the embed template, would it be better to create a new config setting like `uvAppPath` that defaults to `/uv.html`? Then we wouldn't have to change the embed templates as much, and other code looking for the full path to uv.html would also work correctly.

I wonder if it would make sense to split this PR in two: one to do the hash parameter simplification, since I think that is a useful improvement to readability, and one to add the new config setting. Another advantage to this approach: we wouldn't have to change the number of placeholders in the embed template, which would better maintain backward compatibility.

Reply:
That's a good idea. I think that getAppUri should not include /uv.html but if I'm misunderstanding the usage...